pykern/pkcli/ci.py

```diff
@@ -147,3 +136,15 @@ def _paths(cwd):
         _error("no files found")
     if r:
         _error("\n".join(r))
+
+
+def _paths(cwd):
+    if pkio.py_path("setup.py").isfile() and (
```

Comment:
pkio.py_path("setup.py") is better written as cwd.join("setup.py"). All the py.path() can be cwd.

Reply:
3b6bc4ce71f665b83fc9f9cf3024c5434a631d09